### PR TITLE
timelock should now compile

### PR DESCRIPTION
**Pull request by nuel-ikwuoma**

### CODE REVIEW

File: timelock/contracts/Timelock.sol [hunk 1]
```diff
@@ -127,8 +129,8 @@ contract TimeLock {
     }
     
     function changeMaster(address _newMaster) external onlyMaster returns(bool) {
-        require(_keeper != masterKeeper, "");
-        require(_keeper != address(0), "");
+        require(_newMaster != masterKeeper, "");
```
Comment (jonddon) [inline timelock/contracts/Timelock.sol:132]: Add an error message not just empty string, something like "You cannot changer master to masterkeeper"

File: timelock/contracts/Timelock.sol [hunk 2]
```diff
@@ -127,8 +129,8 @@ contract TimeLock {
...
+        require(_newMaster != address(0), "");
```
Comment (jonddon) [inline timelock/contracts/Timelock.sol:133]: Error should be; "You cannot make burn address the master

File: timelock/contracts/Timelock.sol [hunk 3]
```diff
@@ -153,20 +155,19 @@ contract TimeLock {
...
+            lockedSince[msg.sender] = block.timestamp;
+            emit FundsLocked(msg.sender, _amount);
             return true;
         }
         return false;
     }
     
     // get the amount locked up by a project
-    function getLockedAmount(address _projectAddress) external returns(uint) {
+    function getLockedAmount(address _projectAddress) external view returns(uint) {
```
Comment (jonddon) [inline timelock/contracts/Timelock.sol:170]: Why are we locking the token on project Address

File: timelock/contracts/Timelock.sol (hunk 1 above)
Comment (nuel-ikwuoma) [reply timelock/contracts/Timelock.sol:132]: Alright, i would fix that and send another PR

File: timelock/contracts/Timelock.sol (hunk 3 above)
Comment (nuel-ikwuoma) [reply timelock/contracts/Timelock.sol:170]: It makes it easy to identify who(ie which address) has locked what amount of token.
I am open for better suggestion(s) on this.

**Pull request merged**